VWOSdk/Internal/ApiCaller/ApiCaller.cs

```diff
@@ -79,6 +79,11 @@ private static T Deserialize<T>(byte[] byteContent)
             if (byteContent != null)
             {
                 var json = Encoding.UTF8.GetString(byteContent);
+                if (typeof(T) == typeof(Settings)) {
+                    json = json.Replace("\"goals\":{}", "\"goals\":[]");

```

Comment:
a comment would be useful why we need to do this

Reply:
ok